### PR TITLE
Todo List: Interactive list

### PR DESCRIPTION
**Pull request by Megagig**
### **In This Milestone, I:**

1. Added a new JavaScript file and import it as a module:

2. Add event listener to the checkbox (change).

3. Update items object's value for completed key upon user actions.
4. Implemented a function for the "Clear all completed" button.
5. Store the updated array of items in local storage, so the user gets the correct list of values after the page reloads (

### CODE REVIEW

File: src/modules/updatestatus.js [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+//     completed: true,
+//   };
+// };
+
+// export const incompleteTo = (task) => {
+//   return {
+//     ...task,
+//     completed: false,
+//   };
+// };
```
Comment (levy002) [inline src/modules/updatestatus.js]: - [OPTIONAL] Kindly remove the commented code in your codebase in order to keep your repo clean and professional.

File: src/index.js [hunk 2]
```diff
@@ -36,5 +92,71 @@ const renderTodoList = () => {
...
+  saveTodos();
+  renderTodoList();
+  updateTodoIndexes();
+};
+
+const deletecompleted = document.querySelector('p');
+deletecompleted.addEventListener('click', (event) => {
+  event.preventDefault();
+  deletecompletedTodo();
+});
```
Comment (levy002) [inline src/index.js:149]: - [OPTIONAL] Kindly make sure after removing the completed todos, update the `index` of all items in the array and make sure it always starts from `.

File: src/modules/createTodoListItem.js [hunk 3]
```diff
@@ -1,31 +0,0 @@
...
-
-  // Create a horizontal line element
-  const hrEl = document.createElement('hr');
-
-  // Append the horizontal line element to the list item
-  listItemEl.appendChild(hrEl);
-  return listItemEl;
-};
-
-export default createTodoListItemEl;
```
Comment (levy002) [inline src/modules/createTodoListItem.js:31]: - [OPTIONAL] I highly advise you to do not to remove this module, it is always the best and most professional way to divide your code into re-usable modules. Kindly make sure to retrieve this module as it was before.

File: src/modules/Addnew.js [hunk 4]
```diff
@@ -1,9 +0,0 @@
-const addNewTodo = (description, tasks, createTodoListItemEl, todoList) => {
-  const taskIndex = tasks.length;
-  const task = { description, completed: false, index: taskIndex };
-  tasks.push(task);
-  const listItemEl = createTodoListItemEl(task);
-  todoList.appendChild(listItemEl);
-};
-
-export default addNewTodo;
```
Comment (levy002) [inline src/modules/Addnew.js:9]: - [OPTIONAL] I highly advise you to do not to remove this module, it is always the best and most professional way to divide your code into re-usable modules. Kindly make sure to retrieve this module as it was before.

**Review by levy002 — APPROVED** [review]
## Hello @Megagig 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## Status: APPROVED :fire:  :fire: 

## Highlights
- No linter error :heavy_check_mark: 
- The app is working well :heavy_check_mark: 

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: src/modules/updatestatus.js (hunk 1 above)
Comment (Megagig) [reply src/modules/updatestatus.js]: WOW, @levy002 Thank you so much. What a professional and detailed review. i have done this

**Pull request merged**